GoogleForCreators/web-stories-wp
# Redesign: Update layer style panel

## Context

This updates the layer style panel to use the generalized numeric input component. Along with some updates to said component.

## User-facing changes

| Before | After |
|-|-|
| ![Screen Shot 2021-03-02 at 22 17 27](https://user-images.githubusercontent.com/637548/109747436-1e1d7780-7ba5-11eb-87e8-14eef84614b2.png) | ![Screen Shot 2021-03-03 at 09 32 27](https://user-images.githubusercontent.com/637548/109821582-1d67fe00-7c04-11eb-84c7-badfe0ea9ede.png) |

## Testing Instructions

### QA

This PR can be tested by following these steps:

1. Add any element to a page
2. Observe the layer panel to adjust the layer opacity

### UAT




## Checklist




---

Fixes #6469
